sklearn/inspection/_partial_dependence.py

```diff
@@ -532,64 +519,48 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
         Please see :ref:`this note <pdp_method_differences>` for
         differences between the 'brute' and 'recursion' method.
 
+        Note: Ignored by ICE calculation at the moment and 'brute' is used.
+
     n_jobs : int, optional (default=None)
         The number of CPUs to use to compute the partial dependences.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    verbose : int, optional (default=0)
-        Verbose output during PD computations.
+        verbose : int, optional (default=0)
+        Verbose output during partial dependence computations.
 
     fig : Matplotlib figure object, optional (default=None)
         A figure object onto which the plots will be drawn, after the figure
         has been cleared. By default, a new one is created.
 
-        .. deprecated:: 0.22
-           ``fig`` will be removed in 0.24.
-
     line_kw : dict, optional
         Dict with keywords passed to the ``matplotlib.pyplot.plot`` call.
-        For one-way partial dependence plots.
-
-    contour_kw : dict, optional
-        Dict with keywords passed to the ``matplotlib.pyplot.contourf`` call.
-        For two-way partial dependence plots.
+        For ICEs and one-way PDPs.
 
     ax : Matplotlib axes or array-like of Matplotlib axes, default=None
         - If a single axis is passed in, it is treated as a bounding axes
-            and a grid of partial dependence plots will be drawn within
-            these bounds. The `n_cols` parameter controls the number of
-            columns in the grid.
-        - If an array-like of axes are passed in, the partial dependence
-            plots will be drawn directly into these axes.
+            and a grid of PDP/ICEs will be drawn within these bounds.
+            The `n_cols` parameter controls the number of columns in the grid.
+        - If an array-like of axes are passed in, the PDP/ICEs will be drawn
+            directly into these axes.
         - If `None`, a figure and a bounding axes is created and treated
             as the single axes case.
 
-        .. versionadded:: 0.22
+    kwargs : dict
+        Additional keyword arguments passed to the ``calc_method`` and the
+        ``plot`` method of ``display_class``.
+        For ICEs, parameter ``fixed_start_point`` will be included in kwargs.
+        For PDPs, parameter ``contour_kw`` will be included in kwargs.
```

Comment:
Basically, it will remain only `contour_kw` which can be directly given with a default value. It will only have an effect for the PDP plotting but this is fine.

Reply:
Addressed in https://github.com/scikit-learn/scikit-learn/pull/16164/commits/cb54fbd95fefb5a89a6b395c26f11b76ec4f1f9d